code/__DEFINES/access.dm

```diff
@@ -1,148 +1,153 @@
-// Security general access, security records, gulag item storage, secbots
-#define ACCESS_SECURITY 1
+
+/* Access is broken down by department, department special functions/rooms, and departmental roles
+	The first access for the department will always be its general access function
+	Access for departmental roles will start with the head and go down in level of succession
+	If we ever get to a point where we have more departmental roles than the five (four) available slots, we should be looking to make some job access more generic
+	Access goes from Command, Security, Engineering, Medical, Supply, Science, Service, Away Missions, Mech Access, Admin, then Antag
+	Please try to make the strings for any new accesses as close to the name of the define as possible
+	If you are going to add an access to the list, make sure to also add it to its respective region further below
+	If you're varediting on the map, it uses the string. If you're editing the object directly, use the define name*/
+
+/// Command general access, EVA storage windoors, gateway shuters, AI integrity restorer, comms console
+#define ACCESS_COMMAND "command"
+#define ACCESS_AI_UPLOAD "ai upload"
```

Comment:
```suggestion
#define ACCESS_AI_UPLOAD "ai_upload"
```
Why do you use spaces here instead of underscores? Makes it easier to mess up varediting if anything.

Reply:
because it isn't a define and it's easier to hit the spacebar than hit the underscore.

Reply:
![image](https://user-images.githubusercontent.com/34697715/169462649-916ce9a8-8c4c-46df-9ca1-1ceccd24186f.png)